docker-entrypoint.sh

```diff
@@ -2,29 +2,61 @@
 set -e
```

Comment:
`set -euo pipefail` for saftey https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425